### ISSUE TITLE
Add bush tiles

### ISSUE THREAD

**Issue by OcelotWalrus** [OpenHV/OpenHV#1069]
These from OpenTyrian could be used 

![image](https://github.com/user-attachments/assets/0f5cf90f-112b-4e2f-9f9e-218864b28a23)

I just wonder if they'd be used like the woods tiles (passable by tank), or just not passable.

**Comment by WilliamWeberBerrutti** [comment]
Maybe bushes could be passable by tanks, but cannot be destroyed when passed by them (unlike trees) and have speed penalty for all ground units.

**Comment by OcelotWalrus** [comment]
That'd be a nice design idea but rendering units over it would look terrible.

**Comment by WilliamWeberBerrutti** [comment]
What if units were translucent when inside bushes? Or hidden until detected by a detector unit?

**Comment by OcelotWalrus** [comment]
You mean cloaked, similarly to the submarines.

**Comment by WilliamWeberBerrutti** [comment]
For the enemy: Maybe hidden / invisible unless revealed by a detector or when they attack. When revealed, they become translucent.

For the player and allies: translucent.

**Comment by OcelotWalrus** [comment]
So the normal cloak effect then. Maybe only pods could hide in it.

**Comment by WilliamWeberBerrutti** [comment]
We could try hiding both pods and vehicles and see what happens.

**Comment by Dzierzan** [comment]
IMO it is not needed.

**Comment by OcelotWalrus** [comment]
It's really just to have more tiles for map creation. I feel like we need more tiles: maps are getting redundant.

**Comment by WilliamWeberBerrutti** [comment]
> It's really just to have more tiles for map creation. I feel like we need more tiles: maps are getting redundant.

Agreed. This will give more options for ground units. Maybe this bush idea could be extended for water units too? Hiding both pods and vehicles in it seems like a power boost vs air, for example, if there are bushes around the base (not too close from the Main Base).

**Comment by OcelotWalrus** [comment]
Indeed: especially with the new pod woods hiding ability thing. Plus, it'll be a nice addition to make more interesting swampy maps: right now the only thing we can do is add those swamp/acid liquid tiles and make the world lighting darker and greener.

**Comment by Mailaender** [comment]
They sure look very nice, but I am not sure if the ones you picked fit in as in leaf size might be too large for the RTS perspective and I don't know if they tile properly. I think the stealing from Tyrian has also been overdone by now.

**Comment by OcelotWalrus** [comment]
Sure, the tyran tiles were just an example to give an idea of what it'd look like.
